src/windows/new_ride.c

```diff
@@ -793,7 +791,7 @@ static void window_new_ride_scrollpaint(rct_window *w, rct_drawpixelinfo *dpi, i
 			gfx_fill_rect_inset(dpi, x, y, x + 115, y + 115, w->colours[1], 0x80 | flags);
 
 		// Draw ride image with feathered border
-		rideEntry = rideEntries[listItem->entry_index];
+		rideEntry = GET_RIDE_ENTRY(listItem->entry_index);
```

Comment:
Yes. Also fix `window_new_ride_paint_ride_information` and other functions you spot using old way.